src/app/containers/author-form-dialog/index.tsx

```diff
@@ -155,6 +165,34 @@ export const AuthorFormDialog: React.FC<AuthorFormDialogProps> = (props) => {
         value={author.email || ''}
         onChange={handleFormChange}
       />
+      <FormControl variant="outlined" classes={{ root: classes.inputField }}>
+        <InputLabel shrink id="author-affiliations-label">
```

Comment:
Design shows the control as being full width, hence maybe the shrink shouldn't be here (if that's what shrink even does)?

Reply:
Not really, the shrink is for the label to sit at the top. the fullWidth attribute does the trick.